tomwhite/covid-19-uk-data
# Feature/Makefile Automation

# Overview

Introduce make commands to automate the process of gathering the daily UK data. Updated `README.md`.

## Changes

1. Introduced `Makefile` to automate the England daily tasks
1. Updated `tools/extract_totals.py` to append data to `./data/covid-19-totals-uk.csv` if the downloaded temp HTML file contains today's date
1. Updated `.gitignore` to exclude vscode and temp HTML files
1. Updated `README.md` with usage instructions for the make commands

## Steps to test

1. Check out this branch
1. **Make commands**: Follow the instructions in the `Make commands` section of `README.md`
1. **`extract_totals.py`**: Updated usage instructions in `Manually running scripts` to include a second arg for `./data/covid-19-totals-uk.csv`
